### PR TITLE
[AGENT] Fix problems in remote exec

### PR DESCRIPTION
**Pull request by rvql**
- fixed `top` width problem
- fixed `ip address` namespace problem
- moved errmsg to outer message
- added request_id in messages

### This PR is for:
- Agent

#### Affected branches
- main
- 6.5

### CODE REVIEW

File: agent/src/rpc/remote_exec.rs [hunk 1]
```diff
@@ -80,7 +80,7 @@ fn all_supported_commands() -> Vec<Command> {
             command_type: CommandType::Linux,
         },
         Command {
-            cmdline: "top -b -n 1 -c",
+            cmdline: "top -b -n 1 -c -w",
```
Comment (rvql) [inline agent/src/rpc/remote_exec.rs:83]: 不加 `-w` 导致输出宽度限制在 80

File: agent/src/rpc/remote_exec.rs [hunk 2]
```diff
@@ -401,46 +410,31 @@ impl Stream for Responser {
...
+                    request_id: self.result.request_id,
                     command_result: Some(batch),
                     ..Default::default()
                 }));
             }
 
-            if let Some((id, ns_fp, future)) = self.pending_command.as_mut() {
+            if let Some((_, id, future)) = self.pending_command.as_mut() {
                 trace!("poll pending command '{}'", get_cmdline(*id).unwrap());
-                if let Some(f) = ns_fp {
```
Comment (rvql) [inline agent/src/rpc/remote_exec.rs:411]: `setns` 挪到 L652 `command.output()` 周围

**Review by sharang — DISMISSED** [review]

**Review by sharang — DISMISSED** [review]

File: agent/src/rpc/remote_exec.rs [hunk 3]
```diff
@@ -754,3 +837,111 @@ impl fmt::Debug for Params<'_> {
...
+            },
+            Err(_) => {
+                return Err(e.into());
+            }
+        },
+    };
+
+    Ok(Output {
+        status: Default::default(),
+        stdout: serde_json::to_vec_pretty(&dp)?,
```
Comment (rvql) [inline agent/src/rpc/remote_exec.rs:917]: kubectl describe 虽然也有[源码](https://github.com/kubernetes/kubectl/blob/master/pkg/describe/describe.go)，但改了一半感觉意义不大，先 `json` 一把梭了

Comment (sharang) [reply agent/src/rpc/remote_exec.rs:917]: ok。实际上主要是 event，可以考虑只提供 event 都行

**Pull request merged**